event_listener/dfusion_db/tests/snapp_event_receiver_test.py

```diff
@@ -1,12 +1,13 @@
 import unittest
 from unittest.mock import Mock
-from ..snapp_event_receiver import WithdrawRequestReceiver, DepositReceiver, StateTransitionReceiver, \
-    SnappInitializationReceiver, OrderReceiver
+from ..snapp_event_receiver import DepositReceiver, OrderReceiver, SnappInitializationReceiver
+from ..snapp_event_receiver import WithdrawRequestReceiver, StateTransitionReceiver
 from ..models import Deposit, StateTransition, TransitionType, Withdraw, AccountRecord, Order
 
 
 class DepositReceiverTest(unittest.TestCase):
-    def test_writes_deposit(self) -> None:
+    @staticmethod
```

Comment:
Somewhat unrelated changes ☹️

Reply:
Part of unit testing

Reply:
But these tests are not related to this change. You would still want to make this change if this PR was to be reverted.